exercises/scripts/createE2ETestWorkflow.ts

```diff
@@ -0,0 +1,79 @@
+import { argv } from 'process'
+import { writeFile } from 'fs'
+
+const EXERCISES = ['00-init']
```

Comment:
please don't use upper case for some constants, lowercase for others ;) lower-case 'em!

Reply:
Changed to lowercase, see a2853ed